src/controllers/expenceController.js

```diff
@@ -0,0 +1,88 @@
+const expenseService = require('../services/expenceService.js');
+const userService = require('../services/userService.js');
+
+const getExpenses = async (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  res.send(await expenseService.getAllExpenses(userId, categories, from, to));
+};
+
+const getOneExpense = async (req, res) => {
+  const { id } = req.params;
```

Comment:
Consider wrapping the code into a try-catch block and sending 500 errors in the case of db issues